crates/weaver_forge/src/extensions/util.rs

```diff
@@ -131,10 +132,34 @@ pub fn acronym(acronyms: Vec<String>) -> impl Fn(&str) -> String {
     }
 }
 
+// Helper filter to dump value (1st parameter) in yaml format.
+fn to_yaml(value: Value) -> Result<String, minijinja::Error> {
```

Comment:
The function should take a `&Value` and return a `Result<Value, minijinja::Error>`